pkgs/games/teeworlds/default.nix

```diff
@@ -45,17 +52,16 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     python3 lua5_3 zlib
+    wavpack
+  ] ++ lib.optionals stdenv.isDarwin [
+    Cocoa
```

Comment:
If this dependency isn't needed for the server it should be put in the `lib.optionals buildClient` list.